### DISCUSSION TITLE
New to Tailwind: how to style HTML tags which do not exist YET in markup?

### DISCUSSION THREAD

**Discussion by charnould** [tailwindlabs/tailwindcss#13902] [Help] [answered]
Hello,

I'm very new to Tailwind and trying it in a side project (a chatbot) is not-so-simple for me; I'm coming from plain old BEM CSS!

Because I'm using an old tech stack (kind of `Express` for server-side and an old school `script.js` for a very few sprinkles for client-side), I will need to use one (and only one) `@apply`. Let's say it's ok.

But I have a double conceptual problem:

1. My app (via an LLM/AI) generates a stream of text in markdown which is parsed (client-side) in html with super native HTML tags (`h1`, `p`, `li`, `pre`, etc.). Theses tags do not exist in my markup before being LLM-generated... so how may I style them in Tailwind?
2. My Tailwind output is - let's say - a default style. But sometimes I want to modify a few colors of this default theme. How to do so: create a `custom.css` file, loaded after `default.css` containing CSS variables with custom values? Is it the way to go (keep in mind my old tech stack)?

Thanks for your guidance.

**Comment by wongjn** [comment, marked as answer]
1. You could consider using the [typography plugin](https://github.com/tailwindlabs/tailwindcss-typography) for this.
2. For modifying colors, you could:
    - Change the class names used.
    - Alter the Tailwind configuration.
    - Use CSS variables.
  None of these options should facilitate the need for separate stylesheets.

**Reply by charnould** [reply]
Thanks a lot!!
